hibike/devices/BatteryBuzzer/disp_8.cpp

```diff
@@ -0,0 +1,195 @@
+//code that deals with displaying the data to the outside world.
+
+#include <SevenSeg.h>
+#include "disp_8.h"
+
+//Segment Pins, A to G
+SevenSeg disp(A,B,C,D,E,F,G);
+
+const int numOfDigits=4;
+int digitPins[numOfDigits] = {6, 5, 4, 3};
```

Comment:
Shouldn't this be using something from pindefs? (These are pin numbers here, right?)